### ISSUE TITLE
⛔ ❓ Question about Semantic HTML 5

### ISSUE THREAD

**Issue by charliedamon** [elementor/elementor#21067]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Hello.

I'm using Elementor with Hello Elementor theme.

I'm trying to make my page the most semantic possible.

The problem is: The Elementor widgets do not have "HTML tags" for selection (header, footer, article, main, etc).

How you guys make the pages semantic?

The only "way" that I found is add every element inside of container. But that increase the HTML code and the page size.

Exemple:

For a simple header with a logo and a menu I need to add 3 containers.

Container 1 > Header HTML tag
Inside container > Logo element and menu

And what about posts pages?

The main header is in all pages, posts included.

I need to use two header tag? One for the main header of the Website and other for the article?

Exemple:

HEADER tag for entire Website
      LOGO + NAV MENU tag for entire Website

MAIN tag for article
      HEADER tag for article
      ARTICLE tag for article
      FOOTER tag for article

Footer tag for entire Website

Is there a simple way to do this?

Regards.

### Steps to reproduce

N/A

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

N/A

**Comment by m1ga** [comment]
you can use the HTML tag option to do that if I understand your issue correctly:
![Screenshot_20230119_115055](https://user-images.githubusercontent.com/4334997/213423571-40b3988f-7771-4a79-8db8-c65555d66f3c.png)

**Comment by rami-elementor** [comment]
Hello Elementor theme is semantic out-of-the-box. The header uses the `<header>` tag, the footer uses the `<footer>` tag and all the content is wrapper with the `<main>` tag.

When you are using Elementor Pro Theme Builder you have the ability to set custom HTML tags at the document level. In addition, you can set HTML tags at the container level. Elementor provides you the option to create custom semantics by controlling the wrapping HTML tags.

At the widget level, we hard code semantic tags at the widgets level. For example, nav menus use the `<nav>` tag, blockquotes use the `<blockquote>` tag, etc. You can't change the tags at the widgets level, and you should not do that.

**Comment by nicholaszein** [comment]
Hi @charliedamon,

Thank you for reaching out! 🙏 

As @m1ga and @rami-elementor correctly pointed out, 

### ⛔ This is not a bug.

You can select the HTML tag for many of the elements in Elementor, including document tags depending on the type of document you're editing. For example the Header, you can set the document tag to be `<header>`, the same for the footer, and even for Single Templates, when you can set the `<main>` tag. You can learn more about how to edit the Document/Page Settings [here](https://elementor.com/help/document-settings/).

It's important to note that HTML documents should have Only One of these tags:

- `<html>`
- `<head>`
- `<body>`
- `<header>`
- `<main>`
- `<footer>`
- `<h1>`

You should not add more than one of these. So plan ahead before starting to build your layout structure.

Other Elements that can have their tags changed can range from Sections, Columns, Containers, Headings, and Headings inside other Elements (a CTA for example).

For now, that's all we can help with. If you think this is still an issue caused by Elementor, please feel free to open a [new issue report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml) and provide us with the most information you can possibly get by following our guidelines. But keep in mind that this is not the right place to ask this kind of question. These questions are more appropriate to be asked on the Elementor Global Community, or you can check the [Help Center](https://elementor.com/help/) to learn more about what you can do with Elementor.

### ⚠️ Please Note that GitHub issues are reserved for 🐞**Bug Reports**, and should not be used for support.

❔ If you have questions or need general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) on Facebook
- [Elementor Help Center](https://docs.elementor.com)
- [Support Forum](https://wordpress.org/support/plugin/elementor) on WordPress.org 
- [Elementor Developers Documentation](https://developers.elementor.com/)

💎 If you have purchased [Elementor Pro](https://elementor.com/pro), you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

Kind regards

**Comment by rami-elementor** [comment]
@charliedamon 

Read this issue too: #13441
